app/assets/stylesheets/application.css

```diff
@@ -124,7 +135,49 @@ button:active{
   .show-posts-container {
     margin: 20px 40px;
   }
+ 
+  .all-forms {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    padding: 40px;
+    gap: 5px;
+  }
+
+  input {
+    border: 2px solid #000;
+    border-radius: none;
+  }
 
-  .comment-button{
+  .actions {
+    background-color: transparent;
+    /* width: 100%; */
+  }
+
+  #inner-button {
+    border: none;
+    background-color: transparent;
+    /* box-shadow: 0 8px 5px 1px; */
+    /* border: 2px solid #000;   */
+    /* color: #3B3B3B; */
+    /* cursor: pointer;   */
+  }
```

Comment:
- [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧